config.js

```diff
@@ -2,14 +2,13 @@ var config = {}
 
 config.gmail = {}
 config.recaptcha = {}
+config.recaptcha.corpevent = {}
 
 config.gmail.user_name = process.env.GMAIL_USERNAME
```

Comment:
you may also want to move away from username & password towards the GMail OAuth API or the like. means not having to maintain those accounts, and just allowing the app to using the normal endpoints